### ISSUE TITLE
Prevent Phone Auth UI dismissing in iOS 13

### ISSUE THREAD

**Issue by paulomendes** [firebase/FirebaseUI-iOS#787]
Hey Folks, I would like to have a little help in a matter that I'm struggling with.

FirebaseUI for `FUIPhoneAuth` doesn't have a way to prevent the dismiss of the `Auth` flow.

Right now I'm doing this

```swift
guard let authUI = FUIAuth.defaultAuthUI() else { return }
        let providers: [FUIAuthProvider] = [FUIPhoneAuth(authUI: authUI)]
        authUI.shouldHideCancelButton = true
        authUI.providers = providers
        if #available(iOS 13.0, *) {
            authUI.authViewController().isModalInPresentation = true
        }
        guard let phoneProvider = authUI.providers.first as? FUIPhoneAuth else {
            fatalError("Firebase AuthUI always need to have a valid PhoneProvider")
        }
        authUI.delegate = self
        phoneProvider.signIn(withPresenting: navigationController, phoneNumber: nil)
```

I would expect to have the flow of the authentication not being dismissed on iOS 13. But the user can dismiss it with a pan down gesture and this break the intended flow. 

Is there any workaround on this matter?

**Comment by borut-t** [comment]
@paulomendes you can use `isModalInPresentation` flag when presenting UI. I've raised the question for a similar issue here: #784

**Comment by morganchen12** [comment]
This is fixed in 8.2.0.
